ballerina/Module.md

```diff
@@ -103,9 +107,7 @@ connection pool handling.  For its properties and possible values, see the [`sql
 
    E.g., The DB client creation for an H2 database is as follows.
    ```ballerina
-    jdbc:Client|sql:Error dbClient = 
-                               new ("jdbc:h2:~/path/to/database", 
-                                "root", "root");
+    jdbc:Client|sql:Error dbClient = new ("jdbc:h2:~/path/to/database", "root", "root");
```

Comment:
This was formatted as such because code snippets in API docs have a line length restriction. Could you verify this?

Reply:
Adjusted in 2049fed